src/main/java/sg/dex/crypto/Hash.java

```diff
@@ -123,4 +127,34 @@ public static String sha3_256String(String string) {
         return Hex.toString(sha3_256(string));
     }
 
+    /**
```

Comment:
This doesn't make sense as a comment for this function. Document what a function does, not the implementation details.